crates/parser/src/format/character_separated/error_buffer.rs

```diff
@@ -70,7 +70,8 @@ impl<I: Iterator<Item = ParseResult>> ParseErrorBuffer<I> {
     /// Returns true if we successfully added another item to the buffer.
     fn buffer_next(&mut self) -> bool {
         if let Some(item) = self.inner.next() {
-            if item.is_err() {
+            if let Err(err) = item.as_ref() {
+                tracing::warn!(error=err.to_string(), "failed to parse row");
```

Comment:
I'm a little mixed on whether we should log these at this point, because it's possible it could result in a lot of log spam. I think if we keep it, the log should maybe indicate that the error is being ignored.

edit: Now that I think of it, perhaps it'd be best to log these at a debug or info level.

Reply:
Even if the error threshold is set, and we ignore some rows if they fail, is this not a fair warning to the user to say that these rows failed parsing? I think it is

Reply:
Updated the message to explicitly mention that we are ignoring because we are below the threshold

Reply:
> Even if the error threshold is set, and we ignore some rows if they fail, is this not a fair warning to the user to say that these rows failed parsing? I think it is

Yeah, I think that's totally fair ... but I also think that users might set a 50% threshold (I've actually seen this) and then try to parse CSVs with millions of rows and have a bunch of them fail (I've not actually seen this). If we log it at the info level, they'd be shown by default, but at least it'd be easy to turn it off if it became overwhelming. TBH, I'm not super concerned here, so I'm fine with merging as is and re-visiting if we need to.